sdk/core/core/inc/az_span.h

```diff
@@ -282,6 +282,7 @@ AZ_NODISCARD az_result az_span_to_uint32(az_span span, uint32_t* out_number);
  */
 AZ_NODISCARD int32_t az_span_find(az_span source, az_span target);
 
+AZ_NODISCARD int32_t az_span_find2(az_span source, int32_t start, az_span target);
```

Comment:
Did you discuss this with anyone?
Do you think we'll make this public, document it, test, it, make samples for it, and support it in perpetuity?
Do we need 2 find functions? Can we remove 1 of them? I'm not sure what your intention is here with this function.

Reply:
Oh no! please disregard this!

Reply:
Just pushed an update, those changes in SDK/core are gone now. Sorry for that.